code-challenges.test.js

```diff
@@ -13,55 +18,161 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("greaterThanTwo" , () => {
+  const fibLength1 = 6
+  const fibLength2 = 10
+  it ("returns numbers that are greater than 2", () => {
+    expect(greaterThanTwo(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    
+})
+})
+
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// const fibLength1 = 6
+// // Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// const fibLength2 = 10
+// // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+// output after running yarn jest : 
+// ● greaterThanTwo › returns numbers that are greater than 2
+
+//     ReferenceError: greaterThanTwo is not defined
 
 // b) Create the function that makes the test pass.
+// Psuedocode: create a function called greaterThanTwo. This function will take in an input( a number, greater than 2) and return an array which length contains the numbers of the fibonacci sequence.  The greaterThanTwo function will have the parameters of length  of the arrays fibLength1 and FibLength2. This function will take in number > than 2 and returns those numbers in a new array.
+// inputs: fibLength1 and fibLength2
+//outputs: expected outputs [1,1,2,3,5,8] and [1,1,2,3,5,8,13,21,34,55]. I believe I will define an array within the greaterThanTwo function and eshablish a conditonal statement using an for loop to get the expected outputs. I will use the bulit-in methods of .length (property returns the length of a string.) and .push (adds one or more elements to the end of an array and returns the new length of the array.) to itterate over the array and produce the outputs. 
+// (version 1) const greaterThanTwo = (length) => {
+//   let array = [1,1]
+//   for(let i = 2 ; i > length;i++)
+//   array.push(array[i-1]) + array [i+2]
+//   return (array)
+// } 
+// (version 2) 
+const greaterThanTwo = (length) => {
+  let array = [1, 1];
+  for (let i = 2; i < length + 1; i++){
+    array.push(array[i - 2] + array[i -1])
+  }
+ return (array)
+}
+
+//  notes :  I'm still getting failures with this function, I know what I want to do I believe, but I don't know how to exactly express that in the code.  There are gaps in my logic that I'm just not knowledable enough to correct on my own yet. 
```

Comment:
You are soooooooo close. This logic is bringing back 1 number more than the given variables. So 7 and 11 values are returned rather than 6 and 10. To rectify this, I would take a closer look at your push method and what that is doing. Still a Fib Sequence just adding an extra sequence. GREAT WORK!